app/src/main/res/layout/recycler_view_rules_item.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:gravity="center"
+        android:id="@+id/rv_rules_item"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp"
+        android:text=""
+        android:padding="15dp"
+        android:fontFamily="@font/product_sans_regular"
+        android:textAlignment="center"
```

Comment:
"How to play?" must be bold. "Start must be blue". Need add empty lines between sentences, similar on design.